playbooks/common/openshift-cluster/config.yml

```diff
@@ -8,10 +8,10 @@
   tags:
   - always
   gather_facts: no
-  tasks:
-  - include_role:
-      name: openshift_excluder
-      tasks_from: disable
+  roles:
+  - role: openshift_excluder
+    r_openshift_excluder_action: disable
+    r_openshift_excluder_service_type: "{{ openshift.common.service_type }}"
 
```

Comment:
It depends on where the role is used. I am not against combining both where reasonable. Still, I would incline to use role instead or include_role. With the role there is only one place where it can be used so it is easy to find it. Include_role can be basically used anywhere inside `tasks`. (`grep` will discover that but still the code should be self-obvious). We should definitely talk about using (or not using) `include_role` inside a role.